### ISSUE TITLE
Install issue on 91.0 and hassio 2.11

### ISSUE THREAD

**Issue by potgieterdl** [Yoda-x/ha-zha-new#83]
Did my first install of ha-zha-new and seems it cant install the needed python libs.

```
Setup failed for zha_new: Could not install all requirements. Connection lost. Reconnecting…
Unable to install package https://github.com/Yoda-x/zigpy/archive/master.zip#zigpy==100.1.4.6: Failed building wheel for pycryptodome
python-jose-cryptodome 1.3.2 has requirement pycryptodome<3.4.0,>=3.3.1, but you'll have pycryptodome 3.8.1 which is incompatible.
pyrebase4 4.3.0 has requirement oauth2client==4.1.2, but you'll have oauth2client 4.0.0 which is incompatible.
broadlink 0.9 has requirement pycryptodome==3.4.11, but you'll have pycryptodome 3.8.1 which is incompatible.
Command "/usr/local/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-jss16vrl/pycryptodome/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-yl0129vi/install-record.txt --single-version-externally-managed --prefix  --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-jss16vrl/pycryptodome/
```

I used the built in intogration for zha and it worked perfectly, but have xiaomi devices and it doesnt seem to support them. Hence trying your version.

Anything i can try?

**Comment by Adminiuga** [comment]
not a `zha_new` issue. Dependencies are screwed in 0.91.x. should be fixed in 0.91.2 I think may have to do with androidtv component. There were a bunch of issues opened in home-assistant, don't have a link atm

**Comment by Yoda-x** [comment]
thanks @Adminiuga
